src/pages/patientView/therapyRecommendation/MtbTherapyRecommendationTable.tsx

```diff
@@ -58,16 +58,18 @@ export type ITherapyRecommendationProps = {
     oncoKbAvailable: boolean;
     therapyRecommendations: ITherapyRecommendation[];
     containerWidth: number;
-    onDelete: (therapyRecommendation: ITherapyRecommendation) => boolean;
-    onAddOrEdit: (therapyRecommendation?: ITherapyRecommendation) => boolean;
-    onReposition: (
+    onDelete?: (therapyRecommendation: ITherapyRecommendation) => boolean;
+    onAddOrEdit?: (therapyRecommendation?: ITherapyRecommendation) => boolean;
+    onReposition?: (
         therapyRecommendation: ITherapyRecommendation,
         newIndex: number
     ) => boolean;
     oncoKbData?: RemoteData<IOncoKbData | Error | undefined>;
     cnaOncoKbData?: RemoteData<IOncoKbData | Error | undefined>;
     pubMedCache?: PubMedCache;
     isDisabled: boolean;
+    showButtons: boolean;
+    columnVisibility?: { [columnId: string]: boolean };
```

Comment:
Small and easy fix with big result in the UI: this needs to be set in the table component as now it's not used

Reply:
Ah, i missed that this wasn't being used. Fixed and should now properly hide columns.